### PR TITLE
Harshith Data is added to json

### PR DESCRIPTION
**Pull request by Harshi7016**

### CODE REVIEW

File: members/harshithv/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "id": "harshithv",
```
Comment (ankushdharkar) [inline members/harshithv/data.json:2]: ID doesn't match the one in mappings file

**Comment by Harshi7016** [comment]
Id is mapped properly now 😅

**Comment by Harshi7016** [comment]
I have cropped the image as suggested 👍

**Comment by Harshi7016** [comment]
folder name updated with github username

**Comment by Harshi7016** [comment]
I clearly understood the mapping now

**Pull request closed without merging**